lisa/energy_model.py

```diff
@@ -341,6 +341,12 @@ class EnergyModel(Serializable, Loggable):
         distinct estimation for each item in the series.
     """
 
+    _AUTODETECT_ORDER = None
+    """
+    Order in which subclasses are tried when auto-detecting the kind of energy
+    model to load from a target.
+    """
```

Comment:
Do we care about the order ? We really don't expect having kernels with multiple EMs, so we should really only match one. But no big deal.

Reply:
It's mostly to get reproducible results and to speed up the common case, otherwise the order will be random, which is rarely a good idea. Other than that, I agree, the probe functions should only return true for the kernel they handle and False for the others